astropy/modeling/fitting.py

```diff
@@ -2189,12 +2189,8 @@ def populate_entry_points(entry_points):
 
 
 def _populate_ep():
-    # TODO: Exclusively use select when Python minversion is 3.10
     ep = entry_points()
-    if hasattr(ep, "select"):
-        populate_entry_points(ep.select(group="astropy.modeling"))
-    else:
-        populate_entry_points(ep.get("astropy.modeling", []))
+    populate_entry_points(ep.select(group="astropy.modeling"))
 
 
 _populate_ep()
```

Comment:
Can't we just remove the private function entirely? 
```suggestion
populate_entry_points(entry_points().select(group="astropy.modeling"))
```

Reply:
done